contactaccessrights/js/src/access-rights/services/location.js

```diff
@@ -0,0 +1,19 @@
+define([
+  'access-rights/modules/models',
+  'common/services/api'
+], function (models) {
+  'use strict';
+
+  models.factory('api.location', ['api', function (api) {
+    return api.extend({
+      query: function () {
+        return this.sendGET('OptionValue', 'get', {
+          'option_group_name': 'hrjc_location',
+          'json': {
+            sequential: 1
```

Comment:
`sequential: 1` is already defined in [`api`](https://github.com/civicrm/civihr/blob/develop/org.civicrm.reqangular/src/common/services/api.js#L116), I don't think you need to pass `json` at all